code/modules/jobs/job_types/roguetown/garrison/squire.dm

```diff
@@ -20,7 +20,7 @@
 	min_pq = -5 //squires aren't great but they can do some damage
 	max_pq = null
 	round_contrib_points = 2
-	social_rank = SOCIAL_RANK_PEASANT
+	social_rank = SOCIAL_RANK_YEOMAN
```

Comment:
Garrison they may be, they're presently valued worse then MAA, who are yeoman. Squires can be  replaced far more easily- Peasant works fine for them.